types/canonical.go

```diff
@@ -23,32 +23,32 @@ type CanonicalPartSetHeader struct {
 }
 
 type CanonicalProposal struct {
+	Version          uint64                 `json:"version" binary:"fixed64"`
+	Height           int64                  `json:"height" binary:"fixed64"`
+	Round            int64                  `json:"round" binary:"fixed64"`
+	POLRound         int64                  `json:"pol_round" binary:"fixed64"`
+	Timestamp        time.Time              `json:"timestamp"`
 	ChainID          string                 `json:"@chain_id"`
```

Comment:
Should we be consistent with putting the ChainID at the end?

Also, do we even JSON marshal this ever? If not, should we drop the JSON tags too?

Reply:
Good point. Moved to the end. 

> Also, do we even JSON marshal this ever? If not, should we drop the JSON tags too?

No, I kept them in case someone wants to marshal them to JSON though. But yeah, it makes sense to delete those JSON tags and re-add them if we see a need.